bigquery_storage/google/cloud/bigquery_storage_v1beta1/gapic/big_query_storage_client.py

```diff
@@ -142,6 +156,7 @@ def __init__(
                 self.transport = transport(
                     credentials=credentials,
                     default_class=big_query_storage_grpc_transport.BigQueryStorageGrpcTransport,
+                    address=api_endpoint,
```

Comment:
Raises a `TypeError` because the `transport_callable` does not take an `address` argument :

```python
__________________ ERROR at setup of test_create_read_session __________________
mock_transport = <MagicMock spec='BigQueryStorageGrpcTransport' id='140534759968016'>
    @pytest.fixture()
    def client_under_test(mock_transport):
        from google.cloud.bigquery_storage_v1beta1 import client
        # The mock is detected as a callable. By creating a real callable here, the
        # mock can still be used to verify RPCs.
        def transport_callable(credentials=None, default_class=None):
            return mock_transport
>       return client.BigQueryStorageClient(transport=transport_callable)
tests/unit/test_client.py:47:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
self = <google.cloud.bigquery_storage_v1beta1.client.BigQueryStorageClient object at 0x7fd0cc7244d0>
transport = <function transport_callable at 0x7fd0cc7b7c08>, channel = None
credentials = None
client_config = {'interfaces': {'google.cloud.bigquery.storage.v1beta1.BigQueryStorage': {'methods': {'BatchCreateReadSessionStreams':..., 'initial_rpc_timeout_millis': 86400000, 'max_retry_delay_millis': 60000, 'max_rpc_timeout_millis': 86400000, ...}}}}}
client_info = None, client_options = None
    def __init__(
        self,
        transport=None,
        channel=None,
        credentials=None,
        client_config=None,
        client_info=None,
        client_options=None,
    ):
        """Constructor.
        Args:
            transport (Union[~.BigQueryStorageGrpcTransport,
                    Callable[[~.Credentials, type], ~.BigQueryStorageGrpcTransport]): A transport
                instance, responsible for actually making the API calls.
                The default transport uses the gRPC protocol.
                This argument may also be a callable which returns a
                transport instance. Callables will be sent the credentials
                as the first argument and the default transport class as
                the second argument.
            channel (grpc.Channel): DEPRECATED. A ``Channel`` instance
                through which to make calls. This argument is mutually exclusive
                with ``credentials``; providing both will raise an exception.
            credentials (google.auth.credentials.Credentials): The
                authorization credentials to attach to requests. These
                credentials identify this application to the service. If none
                are specified, the client will attempt to ascertain the
                credentials from the environment.
                This argument is mutually exclusive with providing a
                transport instance to ``transport``; doing so will raise
                an exception.
            client_config (dict): DEPRECATED. A dictionary of call options for
                each method. If not specified, the default configuration is used.
            client_info (google.api_core.gapic_v1.client_info.ClientInfo):
                The client info used to send a user-agent string along with
                API requests. If ``None``, then default info will be used.
                Generally, you only need to set this if you're developing
                your own client library.
            client_options (Union[dict, google.api_core.client_options.ClientOptions]):
                Client options used to set user options on the client. API Endpoint
                should be set through client_options.
        """
        # Raise deprecation warnings for things we want to go away.
        if client_config is not None:
            warnings.warn(
                "The `client_config` argument is deprecated.",
                PendingDeprecationWarning,
                stacklevel=2,
            )
        else:
            client_config = big_query_storage_client_config.config
        if channel:
            warnings.warn(
                "The `channel` argument is deprecated; use " "`transport` instead.",
                PendingDeprecationWarning,
                stacklevel=2,
            )
        api_endpoint = self.SERVICE_ADDRESS
        if client_options:
            if type(client_options) == dict:
                client_options = google.api_core.client_options.from_dict(
                    client_options
                )
            if client_options.api_endpoint:
                api_endpoint = client_options.api_endpoint
        # Instantiate the transport.
        # The transport is responsible for handling serialization and
        # deserialization and actually sending data to the service.
        if transport:  # pragma: no cover
            if callable(transport):
                self.transport = transport(
                    credentials=credentials,
                    default_class=big_query_storage_grpc_transport.BigQueryStorageGrpcTransport,
>                   address=api_endpoint,
                )
E               TypeError: transport_callable() got an unexpected keyword argument 'address'
google/cloud/bigquery_storage_v1beta1/gapic/big_query_storage_client.py:159: TypeError
```